src/NumericalAlgorithms/LinearOperators/PartialDerivatives.cpp

```diff
@@ -184,4 +292,29 @@ GENERATE_INSTANTIATIONS(INSTANTIATION, (1, 2, 3),
 GENERATE_INSTANTIATIONS(INSTANTIATION, (1, 2, 3))
 
 #undef INSTANTIATION
+
+#define INSTANTIATION(r, data)                                                \
```

Comment:
perhaps name `INSTANTIATE_SCALAR`? (remember to rename the two occurrences below)